docker/Dockerfile

```diff
@@ -25,6 +26,7 @@ RUN chown -R espressopp:espressopp /home/espressopp/espressopp /home/espressopp/
 USER espressopp
 
 WORKDIR /home/espressopp/espressopp
+RUN maintainer/check_codestyle.sh ${CODESTYLE} ${TRAVIS_BRANCH}
 RUN mkdir build
 
 WORKDIR build
```

Comment:
We could put the block below in a `if [[ $CODESTYLE != ON ]]` to make it fails faster, or at least of that for `make` and `make test`.